### ISSUE TITLE
Accessibility: Form fields for contact data have no label elements

### ISSUE THREAD

**Issue by adkurz** [nextcloud/contacts#4366]
### Describe the bug

Form fields for contact data, such as name or telephone number, have no label elements (<label>). The labels are only contained in simple span elements. Therefore, it is not possible for screen readers and other assistive technologies to link a form field to its name, so the fields do not have an accessible name. This also applies to the read-only form fields in the view of a specific contact.

### Steps to reproduce

1. Add a new contact or edit an existing contact.
2. View a field, e.g. “Name” or “City”.
3. View the HTML code (developer tools in a browser)

### Expected behavior

Any label text should be enclosed in a label element (<label>name</label>). To link a label with the associated form field, the for attribute should be used. The value of this attribute is the form fields ID. Example: For a form field with the id "city" (<input id="city">): <label for="city">City:</label>

### Actual behavior

There are no label elements, only span elements for the label texts.

### Contact version

7.0.1

### Operating system

Debian

### PHP engine version

PHP 8.3

### Web server

Apache (supported)

### Database

MariaDB

**Comment by SebastianKrupinski** [comment]
Hi @adkurz 

Thank you for bringing this to out attention. 

I have made a quick patch, the might address this without converting the tags to labels.

Is there any chance you could test the patch with your accessibility software? 

https://github.com/nextcloud/contacts/pull/4368

**Comment by hamza221** [comment]
shoud be fixed in https://github.com/nextcloud/contacts/pull/4967, @adkurz  please reopen if you think I'm wrong
